src/InteractiveSlide/InstantBook.php

```diff
@@ -315,7 +332,7 @@ function (CacheItemInterface $item) use ($now): \DateTime {
         // Make sure interval is free.
         $busyIntervals = $this->getBusyIntervals($token, [$resource], $start, $startPlusDuration);
         if (count($busyIntervals[$resource]) > 0) {
-            throw new ConflictHttpException('Interval booked already');
+            throw new InteractiveSlideException('Interval booked already', 409);
```

Comment:
```suggestion
            throw new InteractiveSlideConflictException('Interval booked already');
```
Then create `InteractiveSlideConflictException` and map it to 409 Conflict